etc/kayobe/vault.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I've been thinking we should put our defaults in `etc/kayobe/inventory/group_vars/all/vault`, as we do for LVM. This makes it easier to override in etc/kayobe/vault.yml.